Models/model.js

```diff
@@ -0,0 +1,8 @@
+const db = require ("../db/connection.js")
+
+exports.fetchTopics = () => {
+    return db.query(`SELECT * FROM topics`).then((result) => {
+        console.log("line 5")
```

Comment:
Make sure you remove console.logs like this Ben - helpful for setting up and debugging but not relevant as production code